test/Bin/GameTest/GameTests.cpp

```diff
@@ -116,10 +116,6 @@ static auto makeStatusBarTape(TestController *test) {
     return test->tape([] { return engine->_statusBar->get(); });
 }
 
-static auto makeDialogueTypeTape(TestController *test) {
-    return test->tape([] { return uDialogueType; });
```

Comment:
This should still be accessible with a getter for `setDisplayedDialogueType`, no?

Reply:
But dialogue object is not present at all time.

Reply:
Yeah, you can return some placeholder value in this case.

Reply:
Two options here:
1. Split the var. I believe we have c-style downcasts in a couple places, this will also get us rid of them.
2. Just use dynamic_cast.